packages/bot-web-ui/src/components/quick-strategy/form-wrappers/desktop-form-wrapper.tsx

```diff
@@ -0,0 +1,119 @@
+                </div>
+                <div className='qs__head__action'>
+                    <span data-testid='qs-desktop-close-button' onClick={() => handleClose()}>
+                        <Icon icon='IcCloseIconDbot' />
+                    </span>
+                </div>
+            </div>
+            <div className='qs__body'>
+                <div className='qs__body__sidebar'>
+                    <div className='qs__body__sidebar__subtitle'>
+                        <Text size='xs'>{localize('Choose a template below and set your trade parameters.')}</Text>
+                    </div>
+                    <div className='qs__body__sidebar__items'>
+                        <ul>
+                            {(Object.keys(STRATEGIES) as (keyof typeof STRATEGIES)[]).map(key => {
+                                const str = STRATEGIES[key];
+                                const active = key === selected_strategy;
+                                return (
+                                    <li
+                                        className={classNames({ active })}
+                                        key={key}
+                                        onClick={() => onChangeStrategy(String(key))}
+                                    >
+                                        <Text size='xs' weight={active ? 'bold' : 'normal'}>
+                                            {str.label}
+                                        </Text>
+                                    </li>
+                                );
+                            })}
+                        </ul>
+                    </div>
+                </div>
+                <div className='qs__body__content'>
+                    <ThemedScrollbars className='qs__form__container' autohide={false}>
+                        {/* <div className='qs__body__content__head'>
+                            <div className='qs__body__content__head__tabs'>
+                                {FORM_TABS.map(tab => {
+                                    const active = tab.value === active_tab;
+                                    const cs = 'qs__body__content__head__tabs__tab';
+                                    return (
+                                        <span
+                                            className={classNames(cs, { active, disabled: tab?.disabled })}
+                                            key={tab.value}
+                                            onClick={() => setActiveTab(tab.value)}
+                                        >
+                                            <Text size='xs' weight={active ? 'bold' : 'normal'}>
+                                                {tab.label}
+                                            </Text>
+                                        </span>
+                                    );
+                                })}
+                            </div>
+                        </div> */}
```

Comment:
@sandeep-deriv Bhai is this necessary to keep for future use?

Reply:
this is required for the future, once the content is ready we will uncomment it @shafin-deriv